Source/WebCore/css/CSSProperties.json

```diff
@@ -5994,6 +5994,10 @@
                 "uppercase",
                 "lowercase",
                 "full-size-kana",
+                {
+                    "value": "full-width",
+                    "enable-if": "PLATFORM(COCOA)"
```

Comment:
Does this actually make `@supports (text-transform: full-size-kana)` actually false on non-Cocoa platforms?

We should make sure we do what is necessary to make that false on non-Cocoa platforms.

Reply:
That's the intent, although @weinig might know better if this actually works or not.

Reply:
The `PLATFORM(COCOA)` macro doesn't seem to be defined in the parser context. Maybe we should open another PR for this.